### DISCUSSION TITLE
can you help me try and compile

### DISCUSSION THREAD

**Discussion by SkygamerYTYT** [KadeArchive/Kade-Engine#2220] [Q&A]
i cant compile because of this problem Error: rpc_connection.cpp
c:\haxetoolkit\haxe\lib\discord_rpc\git\lib\discord-rpc\src\serialization.h(13): fatal error C1083: Cannot open include file: 'rapidjson/document.h': No such file or directory anyone know how to fix

**Comment by NK-523** [comment]
Try running `haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc` in command prompt.

**Reply by SkygamerYTYT** [reply]
it didnt work but thx
